.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python: 2.7
```

Comment:
sure - it works, its just about beeing explicit here. do you know how travis behaves if somebody i.e. forks it and the repository fork is then newer than 2015-01-01 and uses travis for the fork? I have no idea. it may work or not.